source/process/feature-release.md

```diff
@@ -21,49 +21,54 @@ Pre-work for the current release begins at the code complete date of the previou
 
 ### B. (T-minus 18 working days) Cut-off for submitting features
 
-No pull requests for major features should be **submitted** to the current release after this date. In special cases, exceptions can be made by the Release Manager. 
+No pull requests for major features should be **submitted** to the current release after this date. In special cases, exceptions can be made by the QA Release Manager. 
 
-1. Release Manager:
+1. QA Release Manager:
     - Post this checklist in Release Checklist channel
+2. QA - For each team:
     - Follow that major feature PR reviews are prioritized and post a list of outstanding feature PRs in the Release Discussion channel
     - Check that major features for platform have a corresponding ticket for RN, and vice versa, where applicable
     - Check that all major features are behind a feature flag
     - Confirm with PMs that each Enterprise feature is in the correct [pricing SKU](https://about.mattermost.com/pricing/)
     - Review any features that are currently in beta and confirm with PMs if there are any to be promoted
     - Confirm all config settings and new features have diagnostics
+3. PM - For each team:
+    - Prepare outline for release announcement based on your team's work. This includes an outline, benefits, order of major features, and which features have an accompanying screenshot. 
+    - Review outline with devs and QA in your team.
```

Comment:
This could be moved to T-15 since it's unsure at this point what features will actually make it in.